source/contracts/reporting/Market.sol

```diff
@@ -112,14 +112,14 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
         return true;
     }
 
-    function designatedReport(uint256[] _payoutNumerators) public returns (bool) {
-        // intentionally does not migrate the market as designated report markets won't actually migrate unless a dispute bond has been placed or the designated report doesn't occur
-        require(msg.sender == designatedReporterAddress);
-        require(getReportingState() == ReportingState.DESIGNATED_REPORTING);
-        // we have to create the reporting token so the rest of the system works (winning reporting token must exist)
-        getReportingToken(_payoutNumerators);
+    function designatedReport() public returns (bool) {
+        // Intentionally does not migrate the market as designated report markets won't actually migrate unless a dispute bond has been placed or the designated report doesn't occur
+        // CONSIDER: We are currently implicitly trusting the rporting token to call this at the appropriate state. We may want to add more validation here when the Market contract can afford the extra lines
```

Comment:
Recommend following up on this `CONSIDER` now that we have some breathing room in Market.sol.